### DISCUSSION TITLE
Verification e-mails not received by Agents

### DISCUSSION THREAD

**Discussion by kalamchi75** [chatwoot/chatwoot#4191] [Self hosted]
Hi Guys, 

I have been testing a self-hosted Chatwoot server. 
I'm having an issue receiving the verification e-mail (or reset password e-mail) for agents. 
I have setup and e-mail inbox, and both sending and receiving seem to be working fine. I can receive e-mails in the inbox, and I can reply to them. So the server is communicating correctly with the mail server on ports SMTP 25 and IMAP 143. 
However, none of the agent verification emails have been received (not even in the junk mail). I tried adding my work e-mail and my gmail account, but nothing came to neither. 

Am I missing smth here ?

Does the server use the same e-mail inbox settings for those verification mails ? or is there any other setup  ? 

Below are my .env lines regarding mail senders : 

`    
      # The email from which all outgoing emails are sent
      # could user either  email@yourdomain.com or BrandName <email@yourdomain.com>
      MAILER_SENDER_EMAIL=chatwoot@OUR.DOMAIN

    #SMTP domain key is set up for HELO checking
    SMTP_DOMAIN=mail.OUR.DOMAIN
    # the default value is set "mailhog" and is used by docker-compose for development environments,
    # Set the value as "localhost" or your SMTP address in other environments
    SMTP_ADDRESS=mail.OUR.DOMAIN
    SMTP_PORT=25
    SMTP_USERNAME=chatwoot@OUR.DOMAIN
    SMTP_PASSWORD=SOME-PASSWORD
    # plain,login,cram_md5
    SMTP_AUTHENTICATION=login
    SMTP_ENABLE_STARTTLS_AUTO=false
    # Can be: 'none', 'peer', 'client_once', 'fail_if_no_peer_cert', see http://api.rubyonrails.org/classes/ActionMailer/Base.html
    SMTP_OPENSSL_VERIFY_MODE=none
    # Comment out the following environment variables if required by your SMTP server
    # SMTP_TLS=
    # SMTP_SSL=

    # Mail Incoming
    # This is the domain set for the reply emails when conversation continuity is enabled
    MAILER_INBOUND_EMAIL_DOMAIN=mail.OUR.DOMAIN
`

Thanks

**Comment by sojan-official** [comment]
lets keep track of this over https://github.com/chatwoot/chatwoot/issues/4193
